Generalize ApprovalDenied/RejectionDenied to ParticipationDenied (#115)
upb-uc4/api: hlf/chaincode/contracts/operation.md

    - [GenericError](../errors.md#GenericError) 
      ```json
      {
        "type": "HLParticipationDenied",

>         "title": "You are not allowed to participate in the given operation"

Reviewer: ```suggestion
        "title": "You are not allowed to approve or reject the given operation"
```

    - [GenericError](../errors.md#GenericError) 
      ```json
      {
        "type": "HLParticipationDenied",

>         "title": "You are not allowed to participate in the given operation"

Reviewer: ```suggestion
        "title": "You are not allowed to approve or reject the given operation"
```

    - operationData :: [OperationData](#OperationData) 
      -  Description: Success, returns the list of operations.

    - [GenericError](../errors.md#GenericError) 
      ```json
      {
        "type": "HLParticipationDenied",

>         "title": "You are not allowed to participate in the given operation"

Reviewer: ```suggestion
        "title": "You are not allowed to approve or reject the given operation"
```
